web-rdfs/source/http_handlers.cc

```diff
@@ -228,11 +251,14 @@ void get_handler(std::shared_ptr<HttpsServer::Response> response,
 
   if (!typeOfRequest.compare("OPEN")) {
     read_file_handler(response, path);
+  } else if (!typeOfRequest.compare("LISTST")) {
```

Comment:
type should be LISTSTATUS

Reply:
It only checks part of the request string

Reply:
like it doesn't get the full string?

Reply:
In get_request_type:
std::string typeOfRequest = request->query_string.substr(3, 6);

Reply:
oh forgot about that